scripts/distribution/macOS-package/template/scripts/preinstall

```diff
@@ -34,19 +34,19 @@ function removeLaunchDaemons(){
     removeServiceIfExists "software.concordium.mainnet.node-collector"
     removeServiceIfExists "software.concordium.testnet.node"
     removeServiceIfExists "software.concordium.testnet.node-collector"
-
-    echo "-- Deleting the following files from '/Library/LaunchDaemons/':"
-    removeFileIfExists "/Library/LaunchDaemons/software.concordium.mainnet.node.plist"
-    removeFileIfExists "/Library/LaunchDaemons/software.concordium.mainnet.node-collector.plist"
-    removeFileIfExists "/Library/LaunchDaemons/software.concordium.testnet.node.plist"
-    removeFileIfExists "/Library/LaunchDaemons/software.concordium.testnet.node-collector.plist"
 }
 
-function removeNodeDir(){
-    if [ -d "/Library/Concordium Node/" ]
+# Removes the binaries and report-to-dashboard files if they exist.
+# Does not touch launch daemon files and files created by the user.
+function removeFilesInNodeDir(){
+    local nodeDir="/Library/Concordium Node"
+    if [ -d "$nodeDir" ]
     then
-        echo "-- Deleting '/Library/Concordium Node/'"
-        sudo rm -r "/Library/Concordium Node/"
+        echo "-- Deleting the following files from '$nodeDir':"
+        removeFileIfExists "$nodeDir/concordium-node"
+        removeFileIfExists "$nodeDir/node-collector"
+        removeFileIfExists "$nodeDir/REPORT_TO_NETWORK_DASHBOARD_MAINNET"
```

Comment:
Is this a user configuration? Or what does it mean?
Do we re-create these on every installation? Shouldn't they be untouched if the user do not want 
to report the the network dashboard? 

Alternatively, they could be called  DONT_REPORT_TO_NETWORK_DASHBOARD_MAINNET. 
Then a `blank` installation wouldn't have to create the file and subsequent installations shouldn't bother with adding nor removing these files.

Reply:
If this file exists then the collector is started to report to the network dashboard.

Reply:
Yes, the collector is only started when the file exists.

Reporting to the dashboard is an option in the installer. That's why I remove the old setting, i.e., the old file if it exists.

Reply:
Yes. I forgot it was possible to toggle this behaviour during the installation process.